packages/infra/lib/api-stack.ts

```diff
@@ -326,6 +327,18 @@ export class APIStack extends Stack {
       alarmSnsAction: slackNotification?.alarmAction,
     });
 
+    //-------------------------------------------
+    // Terminology Server Service
+    //-------------------------------------------
+    if (!isSandbox(props.config)) {
+      createTermServerService(
+        this,
+        { ...props, generalBucket },
+        this.vpc,
+        slackNotification?.alarmAction
+      );
```

Comment:
We can't add more resources to the API Stack.

Any particular reason not to have this as its own stack?